File: kildey004/programacaobasica/questao8.py
print ("**Indicador de volumes**")

ldc = float(input("digite o lado do cubo: "))

hprl = float(input("digite a altura do paralelepipedo: "))
pprl = float(input("digite a profundidade do paralelepipedo: "))
cprl = float(input("digite o comprimento do paralelepipedo: "))

bcn = float(input("digite a base do cone: "))
hcn = float(input("digite a altura do cone: "))

res = float(input("digite o raio da esfera: "))

vcubo = ldc ** 3

vprl = hprl * pprl * cprl

vcone = 3.14 * bcn ** 2 * hcn / 3

vesf = 3.14 * res ** 3 / 3

print( " O volume do cubo e: " + str(vcubo))

print( " O vlume do paralelepipedo e: " + str(vprl))

print( " O volume do cone e: " + str(vcone))

print( " o volume da esfera e: " + str(vesf))

File: kildey004/programacaobasica/Questao8.java
import java.util.Scanner;
public class Questao8 {
    public static void main (String[] args) {
        double ldc;
        double pprl;
        double cprl;
        double hprl;
        double bcn;
        double hcn;
        double res; 
        double vcubo;
        double vprl;
        double vcone;
        double vesf;
        Scanner s;

        System.out.println("**Indicador de volumes**");

        s = new Scanner(System.in);
        System.out.println("Digite o lado do cubo:");
        ldc = s.nextDouble();

        System.out.println("Digite a altura do paralelepipedo:");
        hprl = s.nextDouble();

        System.out.println("Digite o comprimento do paralelepipedo");
        cprl = s.nextDouble();

        System.out.println("Digite a profundidade do paralelepipedo:");
        pprl = s.nextDouble();

        System.out.println("Digite a base do cone:");
        bcn = s.nextDouble();

        System.out.println("Digite a altura d cone:");
        hcn = s.nextDouble();

        System.out.println("Digite o raio da esfera:");
        res = s.nextDouble();
                
        vcubo = ldc * ldc * ldc;

        vprl = hprl * pprl * hcn;

        vcone = 3.14 * (bcn * bcn) * hcn / 3;

        vesf = 3.14 * (res * res * res) / 3;

        System.out.println("O volume do cubo eh: " + vcubo);

        System.out.println("o volume do paralelepipedo eh: " + vprl);

        System.out.println("O volume do cone eh: " + vcone);

        System.out.println("O volume da esfera eh: " + vesf);
    }
}
